vmdb/app/controllers/application_controller/ci_processing.rb

```diff
@@ -1112,6 +1112,8 @@ def vm_button_operation(method, display_name, partial_after_single_selection = n
       else
         if request.parameters["controller"] == "service"
           self.send("process_services", vms, method)
+        elsif request.parameters["controller"] == "provider_foreman"
+          send("process_foreman", vms, method) unless vms.empty?
```

Comment:
This doesn't need to be `send("process_foreman")`, you can just do `process_foreman(vms, method)`